torch/csrc/jit/passes/constant_propagation.cpp

```diff
@@ -29,7 +29,11 @@ std::vector<IValue> runNode(Node* n) {
   auto op = getOperation(n);
   Stack stack;
   for (auto input : n->inputs()) {
-    stack.push_back(*(toIValue(input)));
+    if (input->node()->kind() == prim::None) {
+      stack.emplace_back(IValue());
+    } else {
+      stack.push_back(*(toIValue(input)));
```

Comment:
Can't we simply make `toIValue` return an empty `IValue` when called on an output of a `prim::None`? That would make it more robust for all other places that depend on some values being constant.